cardano-cli/test/Test/Cli/Pioneers/Exercise5.hs

```diff
@@ -0,0 +1,65 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Test.Cli.Pioneers.Exercise5
+  ( tests
+  ) where
+
+import           Cardano.Prelude
+import           Hedgehog (Property)
+import           Test.OptParse
+
+import qualified Hedgehog as H
+import qualified Hedgehog.Extras.Test.Base as H
+import qualified Hedgehog.Extras.Test.File as H
+
+-- | 1. We generate a payment signing key
+--   2. We create a tx body
+--   3. We sign the tx body with the generated payment signing key
+prop_createZeroTxOutTransaction :: Property
+prop_createZeroTxOutTransaction = propertyOnce . H.moduleWorkspace "tmp" $ \tempDir -> do
+  -- Key filepaths
+  paymentVerKey <- noteTempFile tempDir "payment-verification-key-file"
+  paymentSignKey <- noteTempFile tempDir "payment-signing-key-file"
+  transactionBodyFile <- noteTempFile tempDir "transaction-body"
+  transactionFile <- noteTempFile tempDir "transaction-file"
+
+  -- Generate payment signing key to sign transaction
+  void $ execCardanoCLI
+    [ "shelley","address","key-gen"
+    , "--verification-key-file", paymentVerKey
+    , "--signing-key-file", paymentSignKey
+    ]
+
+  H.assertFilesExist [paymentVerKey, paymentSignKey]
+
+  -- Create transaction body
+  void $ execCardanoCLI
+    [ "shelley","transaction", "build-raw"
+    , "--tx-in", "91999ea21177b33ebe6b8690724a0c026d410a11ad7521caa350abdafa5394c3#0"
+    , "--tx-out", "addr1v9wmu83pzajplrtpsq6tsqdgwr98x888trpmah2u0ezznsge7del3+0"
```

Comment:
It's good to cover the legacy syntax like this. We should also cover examples allowed by the new syntax like
```
"addr1v9wmu83pzajplrtpsq6tsqdgwr98x888trpmah2u0ezznsge7del3 0 lovelace"
```